document_page_approval/tests/test_document_page_approval.py

```diff
@@ -103,41 +127,153 @@ def test_change_request_from_scratch(self):
         )
 
         self.assertEqual(chreq.state, "draft")
-        self.assertNotEqual(page.content, chreq.content)
-        self.assertNotEqual(page.approved_date, chreq.approved_date)
-        self.assertNotEqual(page.approved_uid, chreq.approved_uid)
 
-        chreq.action_to_approve()
+        # Move change request to 'to approve' state
+        chreq.with_user(self.user2).action_to_approve()
```

Comment:
This is not necessary, it is already in “to approve” status requested by the user who created the history.